### ISSUE TITLE
How to convert GraphQL string into IQueryable

### ISSUE THREAD

**Issue by heavenwing** [graphql-dotnet/graphql-dotnet#3321]
## Summary

Is there an easy way to convert GraphQL string into IQueryable<T>, and I can use it to query data from my custom data source.

**Comment by Shane32** [comment]
Check out https://github.com/SimonCropp/GraphQL.EntityFramework

That’s probably all that’s available within a free library now. I ended up writing my own with more translation capabilities.

**Comment by heavenwing** [comment]
> Check out https://github.com/SimonCropp/GraphQL.EntityFramework
> 
> That’s probably all that’s available within a free library now. I ended up writing my own with more translation capabilities.

@Shane32 Thanks for your reply.
And what's class can I use it in [GraphQL.EntityFramework](https://github.com/SimonCropp/GraphQL.EntityFramework) or this library (graphql-dotnet) ? I just want to translate GraphQL into IQueryable like this way: https://github.com/EntityGraphQL/EntityGraphQL#using-expressions-else-where-eql

**Comment by Shane32** [comment]
I couldn't say.  You'd have to check out the GraphQL.EntityFramework library to see what you can do.

**Comment by sungam3r** [comment]
Agree. I suggest you ask maintainers of https://github.com/SimonCropp/GraphQL.EntityFramework to help with this question.
